src/cell_gater/widgets/scatter_widget.py

```diff
@@ -52,21 +52,36 @@ def __init__(self, model: DataModel, viewer: Viewer) -> None:
         self.marker_selection_dropdown.addItems(self.model.markers)
         self.marker_selection_dropdown.currentTextChanged.connect(self._on_marker_changed)
 
-        # self.scatter_canvas = PlotCanvas()
+        choose_y_axis_label = QLabel("Choose Y-axis")
+        self.choose_y_axis_dropdown = QComboBox()
+        self.choose_y_axis_dropdown.addItems(self.model.regionprops_df.columns)
```

Comment:
```suggestion
        self.choose_y_axis_dropdown.addItems([None] + self.model.regionprops_df.columns)
```
I wouldn't set the area as default. Reason being it is hard coded without any clue of what is in region props. I know most do include area, but users can also have that filtered out or have it spelled in a different manner.

Reply:
so until this is set to a value just create canvas but otherwise plot nothing yet.

Reply:
or just the second column as a default. Any column that is a marker but is not the current marker

Reply:
I understand, I agree with the comment that Area might not be available. 
I think perhaps the best way is to try use Area if it exists otherwise use another channel.